modules/prometheus-exporter_oracle/conf/01-listener.yaml

```
@@ -0,0 +1,18 @@
+module: oracle
+name: "Process listener"
+id: listener
+
+transformation: true
+aggregation: ".mean(by=['hostname'])"
+
+signals:
+  signal:
+    metric: tnslsnr
+
+rules:
+  critical:
+    threshold: 1
+    comparator: "<"
+    lasting_duration: "5m"
+    description: "is not started"
+tip: "the listener process is not started, means that application can not connect to the database"
```

while you aggregate on `hostname` I would say:
```suggestion
tip: "the listener process is not started, means that application can not connect to the database host"
```

---

effectively, listener is related to DBs and works on a host. but oracle listener is to connect on a DB, not on a "DB host", that's why I prefer to keep the suggested tip.

---

ok